pages/account/manage/profile.js

```diff
@@ -227,14 +250,26 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
                   </div>
                 </section>
 
-                <div className="mt-10 border-t border-primary-low-medium/30 pt-6 sm:flex sm:items-center sm:justify-between">
-                  <Button primary={true}>SAVE</Button>
+                <div className="mt-6 flex items-center justify-end gap-x-6">
+                  <Button type="button" onClick={() => setOpen(true)}>
+                    DELETE
```

Comment:
I think this is a good idea, but probably not for this PR, as there are many places this would need to be updated and they could all be updated in a new PR

Reply:
yes please create an issue 👍